packages/codemods/src/transforms/styled-components-to-compiled/__tests__/styled-components-attributes-normalize.test.tsx

```diff
@@ -0,0 +1,413 @@
+import type { API, FileInfo } from 'jscodeshift';
+import transformer from '../styled-components-to-compiled';
+
+jest.disableAutomock();
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const defineInlineTest = require('jscodeshift/dist/testUtils').defineInlineTest;
+
+describe('styled-components attributes normalize', () => {
+  defineInlineTest(
+    { default: transformer, parser: 'tsx' },
+    {
+      plugins: [],
+    },
+    `
+      import styled from 'styled-components';
+
+      export const Input = styled.input.attrs(props => ({
+        style: (props) => ({
+          left: props.left,
+          top: props.top,
+        }),
+      }))\`
+        position: absolute;
+      \`;
+      `,
+    `
+      import { styled } from '@compiled/react';
+
+      export const Input = styled.input\`
+left: $\{props => props.left\};
```

Comment:
that will require adding extra indentations in the transformer's code. do we wanna do it?

Reply:
@MonicaOlejniczak I had a go at getting some simple formatting going here using `pretty-print` functionality. However it made no different because it's a tagged string literal and the printer lib has no context on how `styled-components` should be formatted.

I had a go at formatting it myself but ran into complexity dealing with expressions.